docs/DeveloperGuide.adoc

```diff
@@ -782,11 +782,11 @@ See this https://github.com/se-edu/addressbook-level4/pull/599[PR] for the step-
 
 *Target user profile*:
 
-* tech-savvy individual
+* tech-savvy veterinary practioners
```

Comment:
The term should be Veterinarian / Veterinarian Assistance

Reply:
*practitioners.

Howard's suggestion makes sense too. Because it is more specific and shows clearly who our target audience is.